.agents/skills/crafting-effective-readmes/references/art-of-readme.md

```
@@ -0,0 +1,536 @@
+   d. don't shy away from providing a tiny example of an API function's use if
+      it is not obvious or fully covered in the **Usage** section.
+      However, this can also be a strong signal that the function is too complex
+      and needs to be refactored, broken into smaller functions, or removed
+      altogether
+
+   e. aggressively linkify specialized terminology! In markdown you can keep
+      [footnotes](https://daringfireball.net/projects/markdown/syntax#link) at
+      the bottom of your document, so referring to them several times throughout
+      becomes cheap. Some of my personal preferences on API formatting can be
+      found
+      [here](https://github.com/hackergrrl/common-readme/blob/master/api_formatting.md)
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)